src/api/api.py

```diff
@@ -1 +1,38 @@
-# Flask API for backend
\ No newline at end of file
+from pros import pros
+from flask import Flask, jsonify, abort
+from werkzeug.exceptions import HTTPException
+from flask_cors import CORS
+
+
+api = Flask(__name__)
+
+cors = CORS()
+cors.init_app(api)
+
+
+@api.errorhandler(HTTPException)
```

Comment:
It is a good idea to move this to the end of the file. A good convention for priority is:

1. standard library imports (eg: `import os`)
2. imports from external dependencies (eg: `import flask`)
3. imports from within our codebase (eg: `pros`)
4. module loading code (executed during import itself).
5. public functions (followed by their  private `_helper_func()`s)
6. fallback / catchall functions (followed by their private `_helper_func()` s)


Any code that executes on module loading and not inside a function should typically be at the top of the module (except for driver code `if __name__ == '__main__'` etc, that can be at the bottom). 

The reason is that loading code is kinda like Global context for that module. Anyone seeing that module should know what the context is first. Sometimes, you miss 1 or 2 lines that might be between functions. For eg: `api.register_blueprint(pros)` should be moved to the top

Reply:
I removed the part where it creates a new file, if the file is not found, that can be handled during setup and `FileNotFoundError` will be returned by the calling function

Reply:
Fixed